tensorflow/lite/kernels/unpack_test.cc

```diff
@@ -67,192 +66,97 @@ class UnpackOpModel : public SingleOpModel {
   std::vector<int> outputs_;
 };
 
-// float32 tests.
-TEST(UnpackOpTest, FloatThreeOutputs) {
-  UnpackOpModel<float> model({TensorType_FLOAT32, {3, 2}}, 0);
-  model.SetInput({1, 2, 3, 4, 5, 6});
-  model.Invoke();
+template <typename T>
+void Check(int axis, std::initializer_list<int> input_shape,
```

Comment:
try const & whenever possible, thanks

Reply:
@renjie-liu 

comments taken care.

Reply:
looks like input_shape and expected_output_shape can be const reference as well?